tfjs-core/src/util_base.ts

```diff
@@ -321,7 +323,7 @@ export function repeatedTry(
         reject();
         return;
       }
-      setTimeout(tryFn, nextBackoff);
+      scheduleFn(tryFn, nextBackoff);
```

Comment:
How do you think to always call `setTimeoutCustom(tryFn, nextBackoff)` here? And move `setTimeoutCustom(tryFn, nextBackoff)` to a common place since your current implementation `setTimeoutCustom` is already a common way and not only for `webgl` backend? In this case, you don't need to change `gpgpu_context.ts`?

Reply:
We should replace setTimeout with a custom one only in browser and window is ready. So platform browser may be a good place to host it. 
util_base should not be dependent on any env, but setTimeoutCustom relies on env (for both platform check and flag check). So I'm afraid even with your suggestion, we still need to pass setTimeoutCustom as a parameter (Actually I introduced scheduleFn to avoid the dependency with env).